aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -9,16 +9,21 @@ module aptos_framework::aggregator_v2 {
     /// The value of aggregator underflows (goes below zero). Raised by uncoditional sub() call
     const EAGGREGATOR_UNDERFLOW: u64 = 2;
 
-    /// Element type of the aggregator not supported. Raised by create_aggregator() call,
-    /// if Element type is not supported (currently u64 or u128).
-    const EAGGREGATOR_ELEMENT_TYPE_NOT_SUPPORTED: u64 = 3;
-
     /// The generic type supplied to the aggregator snapshot is not supported.
     const EUNSUPPORTED_AGGREGATOR_SNAPSHOT_TYPE: u64 = 5;
 
     /// The aggregator snapshots feature flag is not enabled.
     const EAGGREGATOR_SNAPSHOTS_NOT_ENABLED: u64 = 6;
 
+    /// The generic type supplied to the aggregator is not supported.
+    const EUNSUPPORTED_AGGREGATOR_TYPE: u64 = 7;
+
+    /// The aggregator identifier is out of bounds (above U64::MAX).
+    const EINVALID_AGGREGATOR_IDENTIFIER: u64 = 8;
+
+    /// The aggregator<u64> read is out of bounds (above U64::MAX).
+    const EREAD_OUT_OF_BOUNDS: u64 = 9;
```

Comment:
these are not something users should ever see. not sure if they should go here, but for now let's just add a comment: "should never happen, indicates issue in the internal implementation" or something

Reply:
@igor-aptos @vusirikala I think the comment here for `EREAD_OUT_OF_BOUNDS` is a bit misleading? We use it for converting `aggregator<u128>` --> `snapshot<u64>`, no? Then it is a cast error and can be user-facing. Why `EREAD_OUT_OF_BOUNDS` would ever happen for `aggregator<u64>`?

Reply:
@georgemitenkov This error is not for casting. I raise this error during the normal expensive read of an Aggregator<u64> when the return value is above u64::MAX. Ideally, this error code should never be raised if our code runs correctly. As we generally prefer raising errors instead of making assertions, I raised this error.

Reply:
Yep yep, was confused at the start :) we should use InvariantViolation then for these things